### PR TITLE
Python wrapper library and benchmark program

### PR DESCRIPTION
**Pull request by jssmith**
We have implemented a Python wrapper for SFS that makes it easy to access EFS from scripts. The implementation here remains partial, but sufficient to demonstrate successful read and write operations. Future steps include full testing, e.g., out-of-bounds and other error conditions, as well as full implementation of the file system APIs (directories, permissions, various metadata).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,3 +153,14 @@ Building and Running NFSv4 Benchmark
 
 `./nfsv4_read_write_test read /some_file_name` to run read benchmark
 
+## Build the Python Benchmarks
+
+1. Ensure you have Python 3.x installed.
+
+2. In `/test` run `make make pysfs.so`
```
Comment (scottnuma) [inline README.md]: Two `make`

File: README.md [hunk 2]
```diff
@@ -153,3 +153,14 @@ Building and Running NFSv4 Benchmark
...
+
+## Run the Python Benchmarks
+
+`python3 python_simply_demo.py --mount-point=$(NFS_SERVER) --test-file=/some_file_name`
+
+`python3 benchmark.py --mount-point=$(NFS_SERVER) --test-file=/some_file_name`
```
Comment (scottnuma) [inline README.md]: Should `$(NFS_SERVER)` be `$NFS_SERVER`

**Review by scottnuma — APPROVED** [review]
`python_simply_demo.py` and `benchmark.py` work on my machine as well.

Minor readme changes

File: README.md (hunk 2 above)
Comment (jssmith) [reply README.md]: yes, or `${NFS_SERVER}`

File: README.md (hunk 1 above)
Comment (jssmith) [reply README.md]: Got it.

**Pull request merged**